src/main/java/com/liferay/faces/util/render/internal/FacesURLEncoderFactoryImpl.java

```diff
@@ -31,8 +31,8 @@
 	// serialVersionUID
 	private static final long serialVersionUID = 6782130515246024964L;
 
-	// Private Data Members
-	private FacesURLEncoder facesURLEncoder;
+	// Private Final Data Members
+	private final FacesURLEncoder facesURLEncoder;
```

Comment:
@stiemannkj1 As I recall, usage of the final keyword for private data members is not commonly used in the codebase.